scss/base/reboot.scss

```diff
@@ -17,6 +17,7 @@ html {
   -webkit-text-size-adjust: 100%;
   -ms-overflow-style: scrollbar;
   -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  cursor: @cursor-url;
```

Comment:
Please move this code to `scss/base/generic.scss` 🙏 
I don't want to change `reboot.scss`, because I copy and use `Bootstrap Reboot v4.1.3`.

Reply:
I've fixed this :slightly_smiling_face: